### PR TITLE
Korean Language Support

### PR DESCRIPTION
**Pull request by mjkim0727**
Some translations missed. but, necessary HUD Translations complete.

### CODE REVIEW

File: src/main/resources/assets/controllable/lang/ko_kr.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    "controllable.button.left_trigger": "왼쪽 트리거",
+    "controllable.button.right_trigger": "오른쪽 트리거",
+    "controllable.button.dpad_up": "위",
+    "controllable.button.dpad_down": "아래",
+    "controllable.button.dpad_left": "왼쪽",
+    "controllable.button.dpad_right": "오른쪽",
+    "controllable.key.nextHotbarItem": "Scroll Hotbar Right",
+    "controllable.key.previousHotbarItem": "Scroll Hotbar Left",
+    "controllable.key.pauseGame": "게임 일시 정지",
+    "controllable.key.previousCreativeTab": "이전 크리에이티브 탭 / 제작법 페이지,
```
Comment (MrCrayfish) [inline src/main/resources/assets/controllable/lang/ko_kr.json]: Missing end quote

File: src/main/resources/assets/controllable/lang/ko_kr.json [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    "controllable.key.nextCreativeTab": "다음 크리에이티브 탭 / 제작법 페이지",
+    "controllable.key.previousRecipeTab": "이전 제작법 탭",
+    "controllable.key.nextRecipeTab": "다음 제작법 탭",
+    "controllable.key.moveDown": "아래로 이동",
+    "controllable.key.moveLeft": "왼쪽으로 이동",
+    "controllable.key.moveRight": "오른쪽으로 이동,
```
Comment (MrCrayfish) [inline src/main/resources/assets/controllable/lang/ko_kr.json]: Missing end quote

**Review by MrCrayfish — CHANGES_REQUESTED** [review]

**Comment by mjkim0727** [comment]
I fixed some syntax errors.

File: src/main/resources/assets/controllable/lang/ko_kr.json (hunk 1 above)
Comment (mjkim0727) [reply src/main/resources/assets/controllable/lang/ko_kr.json]: I fixed it.

**Review by MrCrayfish — APPROVED** [review]

**Comment by MrCrayfish** [comment]
Is this considered ready to merge or are you going to be translating more?

**Comment by mjkim0727** [comment]
Merge now. I translated as possible as my english skill.

**Comment by mjkim0727** [comment]
I ready to merge my commit.

**Comment by MrCrayfish** [comment]
Thank you!

**Comment by MrCrayfish** [comment]
Hey @mjkim0727, I am currently in the process of relicensing Controllable to a more suitable license. I would like your permission to relicense your contributions to Controllable under MIT.

**Pull request merged**